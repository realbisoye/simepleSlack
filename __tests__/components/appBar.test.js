import 'react-native'
import React from 'react'
import Appbar from '../../app/components/appBar'

import renderer from 'react-test-renderer'

it('renders Appbar component correctly', () => {
  const rendered = renderer.create(<Appbar />)
  expect(rendered).toMatchSnapshot()
})
