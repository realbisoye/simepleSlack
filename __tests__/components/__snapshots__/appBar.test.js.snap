// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Appbar component correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#3F0F3F",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "maxHeight": "6%",
      "paddingBottom": 10,
      "paddingHorizontal": 10,
      "top": 0,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
      }
    }
  >
    <View
      accessible={true}
      isTVSelectable={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID="menuButton"
    >
      <Image
        source={
          Object {
            "testUri": "../../../app/assets/slackbot.png",
          }
        }
        style={
          Object {
            "height": 30,
            "width": 30,
          }
        }
      />
    </View>
    <Text
      style={
        Object {
          "color": "#ffffff",
          "fontSize": 18,
          "fontWeight": "500",
          "marginLeft": 10,
          "zIndex": 1000,
        }
      }
      testID="appTitle"
    >
      SimpleSlack
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
      }
    }
  >
    <Icon
      color="#FFFFFF"
      name="search"
      size={25}
    />
    <Icon
      color="#FFFFFF"
      name="more-vert"
      size={25}
    />
  </View>
</View>
`;
